src/nvim/option.c

```diff
@@ -2997,7 +2997,8 @@ getoption_T get_option_value(const char *name, long *numval, char **stringval, i
 //  see SOPT_* in option_defs.h for other flags
 //
 // Possible opt_type values: see SREQ_* in option_defs.h
-int get_option_value_strict(char *name, int64_t *numval, char **stringval, int opt_type, void *from)
+int get_option_value_strict(char *name, int64_t *numval, char **stringval, int opt_type, void *from,
+                            bool alloc)
```

Comment:
all calls with `alloc=true` have `stringval==NULL`, so the `alloc` flag can now be removed.